### PR TITLE
Update jquery strengthify

### PR DESCRIPTION
**Pull request by MorrisJobke**
cc @Xenopathic @DeepDiver1975

### CODE REVIEW

**Comment by MorrisJobke** [comment]
This adds `;` to the end of each JS statement, reorders the variable init and caches some jQuery selections.

**Comment by MorrisJobke** [comment]
@DeepDiver1975 Decide if we want this in 8.0 to reduce maybe problems with the missing `;` or to postpone this to 8.1

File: core/vendor/strengthify/jquery.strengthify.js [hunk 1]
```diff
@@ -111,23 +112,23 @@
...
+					);
 				}
 
 				// reset state for empty string password
 				if(password === '') {
-					$('.strengthify-container').css('width', 0)
+					$container.css('width', 0);
 				}
 
 			})
```
Comment (DeepDiver1975) [inline core/vendor/strengthify/jquery.strengthify.js]: missing ; here

Comment (MorrisJobke) [inline core/vendor/strengthify/jquery.strengthify.js]: Just noticed it too :(

**Comment by MorrisJobke** [comment]
@DeepDiver1975 fixed ;)

**Comment by Xenopathic** [comment]
No obvious problems that I can see, :+1: . I'd suggest waiting for 8.1 for this though - it's new code, and the semicolon problem will be fixed with #13116.

File: core/vendor/strengthify/jquery.strengthify.js [hunk 2]
```diff
@@ -29,22 +29,21 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+/* global jQuery */
 (function ($) {
-	$.fn.strengthify = function(options) {
```
Comment (DeepDiver1975) [inline core/vendor/strengthify/jquery.strengthify.js]: line 5: update version

**Comment by MorrisJobke** [comment]
reopened in #13126

**Pull request closed without merging**